service/controller.go

```diff
@@ -524,6 +575,94 @@ func (s *service) CreateVolume(
 	return nil, status.Errorf(codes.NotFound, "Volume/Filesystem not found after create. %v", err)
 }
 
+func (s *service) createQuota(fsID, path, softLimit, gracePeriod string, size int, isQuotaEnabled bool, systemID string) (string, error) {
+	system, err := s.adminClients[systemID].FindSystem(systemID, "", "")
+	if err != nil {
+		return "", err
+	}
+
+	// enabling quota on FS
+	fs, err := system.GetFileSystemByIDName(fsID, "")
+	if err != nil {
+		Log.Debugf("Find Volume response: %v Error: %v", fs, err)
```

Comment:
don't we return error when the filesystem is not found?

Reply:
updated